app.logorrr.LogoRRR.yml

```diff
@@ -0,0 +1,31 @@
+app-id: app.logorrr.LogoRRR
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: LogoRRR
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+modules:
+  - name: LogoRRR
+    only-arches: [aarch64]
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/LogoRRR/
+      - mkdir -p /app/bin
+      - cp -r binaries/LogoRRR/* /app/lib/LogoRRR/
```

Comment:
use cp -ra here to preserve symlinks.

Reply:
thank you, nice catch. my last commit fixes this.